tools/fragpipe/macros.xml

```diff
@@ -1074,7 +1075,7 @@ $kv
             #if $cxt.prot_norm != 'None'
                 #set $wfdict['tmtintegrator.prot_norm'] = $cxt.prot_norm
             #end if
-            #if cxt.glycan_conditional.use_glycan_composition == 'yes'
+            #if $cxt.glycan_conditional.use_glycan_composition == 'yes'
```

Comment:
Good catch, is there a test missing that could spot this error before?

Reply:
Yes, this line is now covered by the tests - the bigger issue was that the parameter wasn't being evaluated: https://github.com/galaxyproteomics/tools-galaxyp/pull/761/commits/fd36cd446a61315ecedc75a9aea4a64b8ac6ea84#diff-11d9fe8e0e5baad2e54c5f8ca8c0e2c01a1b2d875f21f09ab5d8dc1d35aa7157R327.

I could in the tests set arbitrary non-default parameters from each token macro and check that the change is present in the workflow.